From f3ce4ce09fc3ad5886c8b9714aa61521e11da991 Mon Sep 17 00:00:00 2001
From: Zhang Fuxin <fxzhang@ict.ac.cn>
Date: Thu, 4 Mar 2021 22:59:47 +0800
Subject: [PATCH 1/4] increase capacity and replenishRate to avoid trigger rate
 limit error (it can easily occur for the old setting with web-library)

---
 controllers/ApiController.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/controllers/ApiController.php b/controllers/ApiController.php
index 5c4bf71..d49afd7 100644
--- a/controllers/ApiController.php
+++ b/controllers/ApiController.php
@@ -679,8 +679,8 @@ class ApiController extends Controller {
 			$limits['rate'] = [
 				'logOnly' => false,
 				'bucket' => $this->userID . '_' . $_SERVER['REMOTE_ADDR'],
-				'capacity' => 100,
-				'replenishRate' => 10
+				'capacity' => 1000,
+				'replenishRate' => 100
 			];
 		}
 		// For anonymous request
-- 
2.25.1

